sample-apps/react-native/dogfood/src/components/LiveStream/LiveStreamMediaControls.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  LivestreamAudioControlButton,
+  LivestreamVideoControlButton,
+} from '@stream-io/video-react-native-sdk';
+import React from 'react';
+import { Pressable, StyleSheet, View } from 'react-native';
+import { appTheme } from '../../theme';
+import { Chat } from '../../assets/Chat';
+
+/**
+ * Props for the LivestreamMediaControls component.
+ */
+export type LivestreamMediaControlsProps = {
+  handlePresentModalPress: () => void;
+};
+
+/**
+ * The LivestreamMediaControls component controls the media publish/unpublish for the host's live stream.
+ */
+export const LivestreamMediaControls = ({
+  handlePresentModalPress,
+}: LivestreamMediaControlsProps) => {
+  return (
+    <View style={styles.container}>
+      <LivestreamAudioControlButton />
+      <LivestreamVideoControlButton />
+      <Pressable
+        onPress={handlePresentModalPress}
+        style={[
+          styles.chatContainer,
+          {
+            backgroundColor: appTheme.colors.dark_gray,
+          },
+        ]}
+      >
+        <View style={[styles.icon]}>
+          <Chat color={appTheme.colors.static_white} />
+        </View>
+      </Pressable>
```

Comment:
cleaner to move to a seperate component in a file or atleast in this same file

name it as LiveStreamChatComponent

Reply:
Done 👍🏻